bundler.d/gce.rb

```diff
@@ -1,3 +0,0 @@
-group :gce do
-  gem 'fog-google', '~> 1.14'
```

Comment:
This cause issue while require `google/apis/compute_v1` in `app/models/foreman_google/google_compute.rb`

Reply:
ALso remove app/views/images/form/_gce.html.erb

Reply:
> This cause issue while require

Do we have a workaround for that?